go.mod

```diff
@@ -134,6 +134,7 @@ require (
 	github.com/yvasiyarov/gorelic v0.0.7 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
 	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
+	golang.org/x/mod v0.4.0
```

Comment:
This is pulling in a 4th semver package... can an existing be used?

Reply:
Or we open an issue to get them consolidated

Reply:
I would argue we should consolidate, looking at the other ones in here they are all the same.  I can reuse one but I will have to write a wrapper func to validate them.

Reply:
Going with the official package seems like the best option anyways.